platforms/gke/base/use-cases/federated-learning/examples/nvflare-tff/README.md

```diff
@@ -0,0 +1,281 @@
+
+   ```bash
+   Name                               Status Pods
+   nvflare-client1-57d5b45d84-bmv58   OK     1/1
+   nvflare-client2-895b65d8f-p4fs9    OK     1/1
+   nvflare-server1-66c44ddb47-dhtqz   OK     1/1
+   ```
+
+### Submit the training job
+
+In this section, you submit and run a training job:
+
+1. Open [Cloud Shell](https://cloud.google.com/shell).
+
+1. Configure `kubectl` to access the cluster:
+
+   ```bash
+   gcloud container clusters get-credentials <CLUSTER_NAME> --region <CLUSTER_REGION> --project <CLUSTER_PROJECT> --dns-endpoint
+   ```
+
+   Where:
+
+   - `<CLUSTER_NAME>` is the name of the GKE cluster.
+   - `<CLUSTER_REGION>` is the region of the GKE cluster.
+   - `<CLUSTER_PROJECT>` is the project of the GKE cluster.
+
+1. Get the list of NVIDIA FLARE pods running in the cluster:
+
+   ```bash
+   kubectl get pods --namespace fl-1
+   ```
+
+   The output is similar to the following:
+
+   ```bash
+   NAME                               READY   STATUS             RESTARTS        AGE
+   nvflare-client1-7c5c469cdb-s4xn9   3/3     Running            9 (4m38s ago)   33m
+   nvflare-client2-7b49bdd4d-h9xp5    3/3     Running            9 (4m56s ago)   66m
+   nvflare-server1-9f7cb8dfd-rzl27    3/3     Running            11 (42s ago)    66m
+   ```
+
+1. Open a shell in the NVIDIA FLARE server pod:
+
+   ```bash
+   kubectl exec --stdin --tty --namespace fl-1 <NVIDIA_FLARE_SERVER_POD_NAME> -- /bin/bash
+   ```
+
+   Where:
+
+   - `<NVIDIA_FLARE_SERVER_POD_NAME>` is the name of the NVIDIA FLARE server pod
+
+1. Connect to NVIDIA FLARE:
+
+   ```bash
+   cd /workspace/nvfl/workspace/example_project/prod_00/admin@nvidia.com/startup
+   mkdir -p ../local
+   /bin/bash ./fl_admin.sh
+   ```
+
+   When prompted, the username is `admin@nvidia.com`
+
+   The output is similar to the following:
+
+   ```bash
+   User Name: admin@nvidia.com
+   Trying to obtain server address
+   Obtained server address: server1:8003
+   Trying to login, please wait ...
+   Logged into server at server1:8003 with SSID: ebc6125d-0a56-4688-9b08-355fe9e4d61a
+   Type ? to list commands; type "? cmdName" to show usage of a command.
+   >
+   ```
+
+1. Check that NVIDIA FLARE clients registered with the server:
+
+   ```bash
+   check_status server
+   ```
+
+   The output is similar to the following:
+
+   ```bash
+   Engine status: stopped
+   ---------------------
+   | JOB_ID | APP NAME |
+   ---------------------
+   ---------------------
+   Registered clients: 2
+   ------------------------------------------------
+   | CLIENT | TOKEN    | LAST CONNECT TIME        |
+   ------------------------------------------------
+   | site-2 | REDACTED | Mon Feb 10 13:45:09 2025 |
+   | site-1 | REDACTED | Mon Feb 10 13:45:08 2025 |
+   ------------------------------------------------
+   Done [282012 usecs] 2025-02-10 13:45:11.537354
+   ```
+
+   If you don't see any registered client in the output, wait for a one minute
+   and try again.
+
```

Comment:
You might want to add a step on how to gracefully exit.

Reply:
added